xarray/core/groupby.py

```diff
@@ -220,6 +237,13 @@ def __getitem__(self, key):
     def copy(self, deep: bool = True, data: Any = None):
         raise NotImplementedError
 
+    def as_dataarray(self) -> DataArray:
```

Comment:
```suggestion
    def to_dataarray(self) -> DataArray:
```
to_* is more common, right?